CHANGELOG.md

```diff
@@ -31,6 +31,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
     - Docker installation supported on Linux, MacOS and Windows 10 Pro and higher (recommended way of installing the application)
     - native installation is also supported (recommended on Windows 10 Home and lower)
 - as a rule, using minor versions of docker images (eg. `1.2` or `1.2-alpine`) if possible (@MattCzerner)
+- docs and `docker-compose.yml` templates reflect changes of Docker images in [shopsys/project-base](./project-base/CHANGELOG.md) (@PetrHeinz, @MattCzerner)
```

Comment:
do you really want to link to changelog of project-base here? I mean, it makes sense to me partly but i am not sure, if it was an intention :slightly_smiling_face:

Reply:
It was intentional, there is the description of the changes. You're right that it's kinda weird... Maybe make the whole phrase *changes of Docker images in shopsys/project-base* a link?

Reply:
made whole phrase a link